api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -328,6 +328,8 @@
      */
     int getWindowsFromCheckpoint();
 
+    String getOperatorName();
```

Comment:
In the Jira, @PramodSSImmaneni suggested to have ```getOperatorName()``` which is more specific. I agree with that. It makes it explicit. 
However we can call it ```getName``` and explain in the javadoc which I realize is missing. 
I don't have a strong opinion on this.